test/scaladoc/resources/links.scala

```diff
@@ -38,7 +38,7 @@ package scala.test.scaladoc.links {
 
   /**
    *  Links to the trait:
-   *  - [[scala.test.scaladoc.links.Target$                object Test]]
+   *  - [[scala.test.scaladoc.links.Target$                object Target]]
    *  - [[scala.test                                       package scala.test]]
    *  - [[scala.test.scaladoc.links.Target!.T              trait Target -> type T]]
    *  - [[test.scaladoc.links.Target!.S                    trait Target -> type S]]
```

Comment:
Expand the hidden lines below and look for onlyInObject line 54. The link label doesn't mention onlyInObject.